monitoring/cluster_monitoring/managing-cluster-alerts.adoc

```diff
@@ -13,6 +13,7 @@ include::modules/monitoring-silencing-alerts.adoc[leveloffset=+1]
 include::modules/monitoring-getting-information-about-silences.adoc[leveloffset=+1]
 include::modules/monitoring-editing-silences.adoc[leveloffset=+1]
 include::modules/monitoring-expiring-silences.adoc[leveloffset=+1]
+include::modules/monitoring-known-issues.adoc[leveloffset=+1]
```

Comment:
@maximsvistunov Do you have any thoughts on how to add this information (about removing this alert that is no longer required) to the alerts section? Should we keep it as a known issues section like this? Or do you think we should be more specific and call it "Removing the UsingDeprecatedAPIExtensionsV1Beta1 alerting rule" or something similar?